### PR TITLE
Husky tslint formatter

### PR DESCRIPTION
**Pull request by MiljkovicVeljko**

### CODE REVIEW

File: client-app/src/app/guards/auth.guard.ts [hunk 1]
```diff
@@ -3,14 +3,13 @@ import { CanActivate, Router } from '@angular/router';
 import { AuthService } from '../services/auth.service';
```
Comment (aberonja) [inline client-app/src/app/guards/auth.guard.ts:3]: Why is this file here?

File: client-app/package.json [hunk 2]
```diff
@@ -10,6 +10,12 @@
     "e2e": "ng e2e"
   },
   "private": true,
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged && ng lint",
+      "pre-push": "ng build --aot true"
```
Comment (aberonja) [inline client-app/package.json:16]: Do we need this?

File: client-app/src/app/interceptors/access-token-interceptor.ts [hunk 3]
```diff
@@ -1,26 +1,31 @@
 import { Injectable } from '@angular/core';
```
Comment (aberonja) [inline client-app/src/app/interceptors/access-token-interceptor.ts:1]: Same as above.

File: client-app/src/app/guards/auth.guard.ts (hunk 1 above)
Comment (MiljkovicVeljko) [reply client-app/src/app/guards/auth.guard.ts:3]: Prettier change code format

Comment (aberonja) [reply client-app/src/app/guards/auth.guard.ts:3]: Can we set up prettier just to see files that are actually changed/stagged?

File: client-app/package.json (hunk 2 above)
Comment (MiljkovicVeljko) [reply client-app/package.json:16]: I think we'll need just pre-commit hook for now

File: client-app/src/app/interceptors/access-token-interceptor.ts (hunk 3 above)
Comment (MiljkovicVeljko) [reply client-app/src/app/interceptors/access-token-interceptor.ts:1]: Prettier change code format

**Review by aberonja — APPROVED** [review]

**Review by aberonja — COMMENTED** [review]
Remove obsolete npm packages first. e.g. git-format-staged
